ros/processor/insights_engine_result_consumer.py

```diff
@@ -46,6 +49,7 @@ def __next__(self):
         return msg
 
     def run(self):
+        LOG.info("Processor is up, awaiting msgs ...")
```

Comment:
```suggestion
        LOG.info("%s Processor is running. Awaiting messages.", % self.prefix)
```

Reply:
Just tried this out, I see,

PROCESSING ENGINE **RESULTS** - Processor is running. Awaiting msgs.

Do we want the **RESULTS** part on this notifier log?

Reply:
Any better?

`2021-12-15 22:54:02,489 - INFO  - run - Processor is running. Awaiting msgs.`

Reply:
So, currently, we do have all our logs with such prefixes. IMO, to keep debugging easier, we can use these prefixes so that we are aware the logs are generated from which processor.

Reply:
Set to _Processor is running. Awaiting messages._

Reply:
> So, currently, we do have all our logs with such prefixes. IMO, to keep debugging easier, we can use these prefixes so that we are aware the logs are generated from which processor

Okay, change pushed